libraries/auditd_helper.rb

```diff
@@ -33,14 +33,5 @@ def auditd_rulefile(ruleset = 'audit.rules')
         '/etc/audit/audit.rules'
       end
     end
-
-    def auditd_conffile(conf_file = 'audit.conf')
-      if platform_family?('rhel') && node['platform_version'].to_i >= 6
-        ::File.join('/etc/audit/', conf_file)
```

Comment:
Not sure on the history behind this code, but it may have been intentional to put the rules into a file other than `/etc/audit/audit.rules` since that file may be autogenerated (your changes discarded) by augenrules.
I'm unsure because the correct path for input files to augenrules is `/etc/audit/rules.d` and that's not what this code was doing.

Reply:
Nevermind, `auditd_rulefile` does what I was describing.

@rmoles - git blame says you added this code for https://github.com/chef-cookbooks/auditd/commit/0fafd6895be32c8acfd8700efa1c8abb507af869 - can you give a +1 to this part of the changes?

Reply:
IIRC I wrote that to give future users the option to pass in a different file name if they wished. However I'm happy enough for it to be removed.